store/pom.xml

```diff
@@ -65,5 +65,10 @@
             <artifactId>slf4j-api</artifactId>
```

Comment:
Delete this dependency and do not add `logback-classic`

Reply:
> Delete this dependency and do not add `logback-classic`

why？ The test method is not working properly。

Reply:
refer to https://github.com/apache/rocketmq/issues/5890#issuecomment-1384779484